README.md

```diff
@@ -223,3 +164,29 @@ $ score-k8s resources get-outputs 'dns.default#demo-app.dns' --format '{{.host}}
 ### Once I have provisioned a resource, how do I delete it or clean it up?
 
 Resource cleanup has not been implemented yet. The only mechanism today is limited to deleting the Kubernetes manifests output by a template provisioner. As a workaround, the YAML structure in `.score-k8s/state.yaml` can be interpreted to determine what side effects need to be cleaned up.
+
+## Get in touch
+
+Connect with us through the [Score Slack channel](https://join.slack.com/t/scorecommunity/shared_invite/zt-2a0x563j7-i1vZOK2Yg2o4TwCM1irIuA) or contact us via email at team@score.dev.
+
+We host regular community meetings to discuss updates, share ideas, and collaborate. Here are the details:
+
+| Community call | Info |
```

Comment:
Don't duplicate this content here. Instead indicate that this information can be found in the main score readme.